README.md

```diff
@@ -12,7 +12,7 @@
 
 
 -	[Terraform](https://www.terraform.io/downloads.html) 0.10.x
--	[Go](https://golang.org/doc/install) 1.11.x (to build the provider plugin)
+-	[Go](https://golang.org/doc/install) 1.14.x (to build the provider plugin)
```

Comment:
@jrhouston I noticed the Go version hadn't been updated in a while... is this ok to do? Or should I make it a separate PR? I updated it as part of some troubleshooting, but it's not necessarily needed for this PR.

Reply:
This is fine!